jest.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   },
   moduleNameMapper: {
     ".+\\.(css|styl|less|sass|scss)$": `identity-obj-proxy`,
-    ".+\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": `<rootDir>/__mocks__/file-mock.js`,
+    ".+\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": `<rootDir>/src/tests/mocks/file-mock.js`,
```

Comment:
hmmm. Do you know what that file mock is for? Default file?

Usually how I test components etc is the test is in the same folder as the component/whatever I want to test. Not having it the rails way in a separate folder `tests`.

Reply:
We can keep them like that. Just wanted to share how I have seen them

Reply:
Gatsby recommends a mock of itself, for links and the like, in the unit tests.